app/models/orders.rb

```diff
@@ -0,0 +1,5 @@
+class Orders < ApplicationRecord
```

Comment:
tên class model để số ít nhé
```suggestion
class Order < ApplicationRecord
```